src/main/java/ch/njol/skript/conditions/CondIsWhitelisted.java

```diff
@@ -19,72 +19,72 @@
 package ch.njol.skript.conditions;
 
 import org.bukkit.Bukkit;
-import org.bukkit.entity.Player;
+import org.bukkit.OfflinePlayer;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 import ch.njol.skript.Skript;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Condition;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 
 @Name("Is Whitelisted")
-@Description("Whether or not the server or a player is whitelisted.")
-@Examples({"if server is whitelisted:", "if player is whitelisted"})
-@Since("2.5.2")
+@Description("Whether or not the server or a player is whitelisted, and server is whitelist enforced.")
+@Examples({
+	"if player is whitelisted:",
+	"if server is whitelisted:",
+	"if server is whitelist enforced:"
+})
+@Since("2.5.2, INSERT VERSION (enforce)")
+@RequiredPlugins("Minecraft 1.17+")
```

Comment:
Is the 1.17+ for the enforce? if so this change can be applied 
```suggestion
@RequiredPlugins("Minecraft 1.17+ (enforce)")
```

Reply:
Yes, it is 1.17. Made an oopsie in the PR description 😅.